Cargo.lock

```diff
@@ -687,9 +674,9 @@ dependencies = [
 
 [[package]]
 name = "cc"
-version = "1.0.58"
+version = "1.0.63"
```

Comment:
it seems that upgrade of `cc` caused some issues https://github.com/rust-rocksdb/rust-rocksdb/issues/474#issuecomment-730377875

Reply:
Thanks for the pointer @ordian!

Downgrading to `v1.0.62` seems to compile fine on the build machine. Let's see if the same is the case for our pipeline steps or whether I need to do a fresh recompile to trigger the failure.

Reply:
https://github.com/paritytech/substrate/issues/7522